samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1058,6 +921,334 @@ or
+
+            Where the :file:`.pem` files are the self-signed CA certificate and private key files :ref:`you created <nrf_cloud_multi_service_create_device_cred_locally>`.
+
+      .. group-tab:: CoAP
+
+            .. parsed-literal::
+               :class: highlight
+
+               python3 device_credentials_installer.py --ca self\_\ *self_cert_serial*\ \_ca.pem --ca_key self\_\ *self_cert_serial*\ \_prv.pem --id_str nrf- --id_imei -s -d --coap
+
+            Where the :file:`.pem` files are the self-signed CA certificate and private key files :ref:`you created <nrf_cloud_multi_service_create_device_cred_locally>`.
+            The ``--coap`` option indicates that the device needs the CoAP root CA installed.
+            See below for more details.
+
+   .. note::
+      This command assumes you have left the :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_SRC_IMEI` option enabled and the :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX` option set to ``nrf-``.
+      See :ref:`configuration_device_id` to use other device ID formats.
+
+   This script generates, signs, and installs a device credential for your device.
+   The private key for this credential is generated by the device itself, and stored on the modem.
+
+   This script also installs any nRF Cloud root CA certificates required in a single chain to the :kconfig:option:`CONFIG_NRF_CLOUD_SEC_TAG` security tag (``sec_tag``).
+   CoAP connections use one root CA certificate, whereas HTTPS and MQTT use another.
+   Devices using CoAP need both installed, since HTTPS is used for FOTA and PGPS on CoAP devices.
+
+   If the script succeeds, you should see the following output:
+
+   .. code-block:: console
+
+      Saving provisioning endpoint CSV file provision.csv...
+      Provisioning CSV file saved
+
+   And a new file, :file:`provision.csv` should be created.
+   This file will be used in the next step.
+
+#. Navigate to the `Bulk Onboard Devices`_ page of the nRF Cloud portal and upload the :file:`provision.csv` file to onboard the device.
+
+   To get there from the :guilabel:`Dashboard`, click :guilabel:`Devices` under :guilabel:`Device Management` in the navigation pane on the left, then click :guilabel:`Add Devices` and select **Bulk Onboard**.
+
+   Once the `Bulk Onboard Devices`_ page is open, drag in the :file:`provision.csv` file and click **Onboard**.
+
+   You should see a message stating that the file was uploaded successfully, and your device should appear in the `Devices <nRF Cloud Portal Devices_>`_ page.
+
+.. _nrf_cloud_multi_service_provisioning_service:
+
+Provisioning with the nRF Cloud Provisioning Service
+====================================================
+
+If you have :ref:`enabled support <nrf_cloud_multi_service_building_provisioning_service>` for the `nRF Cloud Provisioning Service`_, you can provision and onboard your device as follows:
```

Comment:
"you can instead provision and onboard..." might be better

Reply:
I worry that might be a bit confusing for anyone who jumps to this section from a cross-linking section

Reply:
then the start of the section describing the various methods should mention that the customer only needs to do one of them, not all of them

Reply:
It does, doesn't it?

See the disambiguation on line 930:

```
The following sections outline the various onboarding methods that this sample supports.
  * If you are building with :ref:`LTE connectivity <nrf_cloud_multi_service_building_lte>` (without :ref:`provisioning service support <nrf_cloud_multi_service_building_provisioning_service>`), start with :ref:`nrf_cloud_multi_service_lte_onboarding`.
  * If you are building with :ref:`provisioning service support <nrf_cloud_multi_service_building_provisioning_service>`, start with :ref:`nrf_cloud_multi_service_provisioning_service`.
* If you are :ref:`building with Wi-Fi connectivity <nrf_cloud_multi_service_building_wifi_conn>`, start with :ref:`nrf_cloud_multi_service_onboard_hardcoded`.
```

Reply:
I could make it more explicit by adding a note:

`Note that you only need to perform one of these methods, select the one that best matches your build configuration`